src/app/campaign/campaign.component.ts

```diff
@@ -67,14 +64,18 @@ export class CampaignComponent implements OnInit, OnDestroy {
   campaignLoaded$: Observable<boolean>;
   campaignLoading$: Observable<boolean>;
   campaignSaving$: Observable<boolean>;
-  error$: Observable<any>;
   valid$: Observable<boolean>;
   changed$: Observable<boolean>;
   campaignName$: Observable<string>;
   campaignActualCount$: Observable<number>;
   routeSub: Subscription;
 
-  constructor(private route: ActivatedRoute, private store: Store<any>, private campaignActionService: CampaignActionService) {}
+  constructor(
+    private route: ActivatedRoute,
+    private store: Store<any>,
+    private campaignActionService: CampaignActionService,
+    private campaignErrorService: CampaignErrorService
```

Comment:
The error service is injected into the main parent component of the campaign/flights forms to instantiate the service and subscribe to the error selector.